addons/rose/app/styles/rose/components/header/_index.scss

```diff
@@ -10,7 +10,6 @@
 @use 'brand';
 @use 'nav';
 @use 'utilities';
-@use 'dropdown';
```

Comment:
Did you confirm this wasn't being used? Have you tried running your branch locally? This is what it looks like for me:
<img width="1392" alt="image" src="https://github.com/user-attachments/assets/f4939bd1-e836-4962-8e06-066a04f62b30" />